src/index.js

```diff
@@ -0,0 +1,63 @@
+// Import CSS file
+import './style.css';
+
+// Get the container where tasks will be displayed
+const tasksDiv = document.querySelector('.to-do-list');
+
+// Array of tasks to be displayed
+const arrayOfTassks = [
+  { description: 'task 1', completed: false, index: 0 },
+  { description: 'task 2', completed: false, index: 1 },
+  { description: 'task 3', completed: false, index: 2 },
+  { description: 'task 4', completed: false, index: 3 },
+];
+
+// Function to display tasks
+const displayTasks = () => {
+// Clear any existing tasks in the container
+  tasksDiv.innerHTML = '';
+
+  // Loop through array of tasks and create a div for each one
+  arrayOfTassks.forEach((task) => {
+    // Create a div element for each task
+    const div = document.createElement('div');
+    div.className = 'task';
+    div.setAttribute('data-id', task.index);
+
+    // Create a check button for each task
+    const check = document.createElement('span');
+    check.className = 'check';
+    check.innerHTML = '<i class="fa-regular fa-square"></i>';
```

Comment:
- [ Required ] I think it would be better if you used an input of type `checkbox` instead of an icon, this will make your design match the wireframe provided in the video for this project. Currently, when I open your application on the left-hand side of the descriptions there are no `checkboxes` please consider adding these checkboxes to match the design.

##  Your design
![Screenshot from 2023-03-01 20-24-59](https://user-images.githubusercontent.com/16392046/222216945-02b3eb50-3dfc-4c2d-8761-d4ae36f56737.png)

## Project Video
https://www.youtube.com/watch?v=AcUd-_Yjjqg